e2e-tests/custom-login/conf.js

```diff
@@ -0,0 +1,65 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/* eslint import/no-unresolved:0, import/no-extraneous-dependencies:0, no-console:0 */
+/* global jasmine */
+const jasmineReporters = require('jasmine-reporters');
+const daemonUtil = require('../tools/daemon-util');
+
+const promises = Promise.all([
+  daemonUtil.startCustomLoginServer()
+]);
+
+const config = {
+  // Update these parameters to match your environment for tests
+  params: {
+    login: {
+      username: 'george',
```

Comment:
I wonder if we should add some conditional logic here and inject environment variables. Specifically, I'm thinking about:
- Someone wants to run the tests downstream in the sample apps with their own org + env vars
- We decide to encrypt environment variables for enhanced security

Reply:
As mentioned in other comments, all config should go up to config.js and be overridable by enviroment vars